### DISCUSSION TITLE
Did remix supports dynamic rewrite pathname when serve request

### DISCUSSION THREAD

**Discussion by CreateSun** [remix-run/remix#10337] [Q&A]
Did i can through custom express server modify req.url?

My goal:
When the basename in package.json is '/home',  the app will serve app/_index.tsx when i visit '/main'  path

**Comment by sergiodxa** [comment]
Remix has not support for rewrites because RR doesn't, even if you do the rewrite server-side it won't work once the app is hydrated client-side.
